server/src/test/java/org/elasticsearch/action/admin/cluster/stats/MappingStatsTests.java

```diff
@@ -308,7 +308,7 @@ public void testChecksForCancellation() {
             .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 4)
             .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 1)
             .build();
-        IndexMetadata.Builder indexMetadata = new IndexMetadata.Builder("foo").settings(settings);
+        IndexMetadata.Builder indexMetadata = new IndexMetadata.Builder("foo").settings(settings).putMapping("{}");
```

Comment:
Can we add this randomly instead?

Reply:
I had to add this because we never execute the check if there's no mapping with these changes.